### ISSUE TITLE
Existing FL - Map Tiles Not Loading

### ISSUE THREAD

**Issue by rickleegit** [department-of-veterans-affairs/vets-website#1325]
@ayaleloehr: Map tiles are not rendering this morning. I have tried to load on and off VPN; in Chrome and Safari with the same behavior. Screenshot attached. Can you reproduce this error?

<img width="1438" alt="screen shot 2016-03-07 at 6 45 27 am" src="https://cloud.githubusercontent.com/assets/11138830/13568622/7e969040-e430-11e5-8683-5bf9537beef4.png">

**Comment by ayaleloehr** [comment]
I can reproduce. The code has not been changed recently. Looking at the developer console, it also seems the Google Maps tiles are being returned correctly (not an API error).  The error is `Uncaught TypeError: Cannot read property 'querySelector' of null` coming from `/assets/js/vendor/menu.js`

```
$(document).ready(function() {
	var triggerBttn = document.getElementById( 'trigger-overlay' ),
		overlay = document.querySelector( 'div.overlay' ),
		closeBttn = overlay.querySelector( 'button.overlay-close' );
```
@awong-dev could this be related to the JS bundle errors last week?

**Comment by awong-dev** [comment]
not bundling per se...but likely related.

It can't find div.overlay.

**Comment by awong-dev** [comment]
will need to talk with you tomorrow. I don't understand this page's structure enough to fix it.

**Comment by ayaleloehr** [comment]
I don't believe there is a div.overlay. Looks to me like that menu.js file is from a newer commit to "Move healthcare-app JS into webpack" but it's still being loaded when the facility locator page is loaded?  Ping me when you have time for this today @awong-dev and we'll work through it. Thanks!

**Comment by ayaleloehr** [comment]
Closed via 2dd6c46eea4ff5c708592783d9294d00cb2806bb and a push to production.
